src/inject.js

```diff
@@ -136,6 +136,9 @@ const stringHandlers = {
 // set of class names)
 let alreadyInjected = {};
 
+// The number of classes that have been injected
+let classUIDCount = 0;
```

Comment:
Makes sense. What do you think about hashing on the combine className?

ie this value:

```
processedStyleDefinitions.classNameBits.join("-o_O-")
```